src/Component/Filter/AttributeCombo/AttributeCombo.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+
+import { Select } from 'antd';
+import FormItem from 'antd/lib/form/FormItem';
+const Option = Select.Option;
+
+/**
+ * Combobox offering the attributes to be filtered on.
+ */
+class AttributeCombo extends React.Component<any, any> {
+
+  // PROPS
+
+  /** the internal data object (definition structure) */
+  internalDataDef: any;
+
+  /** label  for this combobox */
+  label: string = 'Attribute';
+
+  /** placeholder shown when combobox has no selection */
+  placeholder: string = 'Select Attribute';
+
+  /** Handler function bound to the 'onChange' of the underlying Select */
+  onAttributeChange: Function;
+
+  constructor(props: any) {
```

Comment:
If you don't perform any actrions with props or the state you can omit the constructor.

Reply:
Agree...